### PR TITLE
WIP: kvmtool: Initial commit

### PR DESCRIPTION
**Pull request by temerkhanov**
- Add initial kvmtool version
  
Signed-off-by: Sergey Temerkhanov <s.temerkhanov@gmail.com>

### CODE REVIEW

File: pkg/dom0-ztools/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
+
 RUN mkdir -p /out/etc/apk /out/boot && cp -r /etc/apk/* /out/etc/apk/
 RUN apk add --no-cache --initdb -p /out \
     zfs=0.8.4-r0
 
+
 FROM scratch
 COPY --from=zfs /out/ /
-ADD rootfs/ /
+ADD rootfs.tar /out/
```
Comment (rvs) [inline pkg/dom0-ztools/Dockerfile:10]: It seems to me that this rootfs.tar is making things more complicated -- unless there's a good reason -- I'd rather avoid doing this and revert the rest of the changes to dom0-ztools

**Review by rvs — CHANGES_REQUESTED** [review]
It looks pretty reasonable @temerkhanov -- but I really don't think we should be making that rootfs.tar unless there's a good reason.

Also, we need to setup dockerHUB repo for this new container

File: pkg/dom0-ztools/Dockerfile (hunk 1 above)
Comment (temerkhanov) [reply pkg/dom0-ztools/Dockerfile]: Dropped the commit

**Review by rvs — CHANGES_REQUESTED** [review]
Hey @temerkhanov -- thanks for taking care of rootfs.tar -- sorry but I forgot to ask one more thing: wouldn't it make more sense to call this one `kvm-tools` to be consistent with our `xen-tools` naming scheme? Also, can you please tell me what's the difference between rootfs.img built with and without this additional container? (I just want to make sure we're not bloating image too much)

**Comment by rvs** [comment]
Hey @temerkhanov -- do you think we can close this one in favor of #1662 (which is based on this work of yours but also does the rename + adds microvm qemu)

**Comment by temerkhanov** [comment]
Included in kvm-tools

**Pull request closed without merging**